examples/native-glutin/Cargo.toml

```diff
@@ -7,3 +7,5 @@ edition = "2018"
 [dependencies]
 glutin = "0.19.0"
 gl = "0.11.0"
+# wasm feature is default - exclude them for quicker compile
+yage = { path = "../../", features = ["native", "gltf"], default-features = false }
```

Comment:
Default features are a bit tricky. The main crate specifies `default = ["wasm", "gltf"]`, so `wasm` is always active unless `default-features = false` is set explicitly. It also compiles fine without it (implicit `default-features = false`), but uselessly includes web-sys and js-sys.
An alternative would be to make `native` the default feature - but then some commands get more complicated, e.g. `wasm-pack build -- --features wasm` (would fail without specifying the feature).

Hints: 
https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html